### DISCUSSION TITLE
no addPrintPageOption available

### DISCUSSION THREAD

**Discussion by langert** [pimcore/pimcore#15121] [Q&A / Support]
Hi,
I set up the web2PrintSettings with WkHtmlToPdf and the testpage iopend from the setttings page is shown without error. Nice!
But there is no add PrintPage option in the document tree? Why? Is there any more configuration needed? Nothing found in the docs => https://pimcore.com/docs/pimcore/current/Development_Documentation/Documents/Document_Types/Print_Documents.html ?

**Comment by kingjia90** [comment]
Just to make sure, did you mark this checkbox? 
![image](https://user-images.githubusercontent.com/6014195/236515294-7bb13665-3e49-43ba-92fd-043b47ec8122.png)

**Reply by langert** [reply]
yes, the option is active

**Reply by kingjia90** [reply]
then maybe check if the perspective item `document.addBlankPrintPage` is properly set, and if there's a js error in the console. 
https://github.com/pimcore/pimcore/blob/e8dbc4da58ae86618bceb67ed35ce23e5e54d2ed/bundles/AdminBundle/Resources/public/js/pimcore/document/tree.js#L400C21-L439
